e2e/tests/regression/test_basic.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Basic Test Cases
+
+Test Tags    regression
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/snapshot.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Volume Basic Test
+    [Tags]  coretest
+    [Documentation]    Test basic volume operations
+    ...
+    ...                1. Check volume name and parameter
+    ...                2. Create a volume and attach to the current node, then check volume states
+    ...                3. Check soft anti-affinity rule
+    ...                4. Write then read back to check volume data
+    When Create volume wrong_volume-name-1.0
```

Comment:
Please follow the BDD style, adopting the Given, When, Then structure.

https://github.com/longhorn/longhorn/issues/8375

Reply:
Refined. Thank you!